LAVA_setup_guide.md

```diff
@@ -0,0 +1,172 @@
+    * search for SITE_ID and change it to SITE_ID = 2
+    * `sudo service lava-server-gunicorn restart`
+    * any time you change site domain you have to change the index to match
+
+#### Add Worker
+
+- `sudo lava-server manage workers add <WORKER_NAME>`
+    * don't do that on a single machine install, it will break lava
+    * set the hostname to the worker name
+- copy the token string to be used when configuring the worker
+
+#### Add device definition
+
+- create a new file in /etc/lava-server/dispatcher-config/devices with the name of the device (you can use the one from https://github.com/paul-szczepanek-arm/lava-prototype)
+    * You can just copy the whole directory https://github.com/paul-szczepanek-arm/lava-prototype/tree/master/lava-server to /etc/, otherwise follow steps below
+    * must have extension jinja2: `mbed.jinja2`
+    * the device should inherit from a device type in /usr/share/lava-server/device-types, to inherit from docker add line: `{% extends "docker.jinja2" %}`
+    * Must pass in extra parameters to docker: `{% set docker_extra_arguments = ["-v /dev:/dev --group-add=dialout --device-cgroup-rule 'a 166:* rwm'"] %}`
+- `sudo service lava-server-gunicorn restart`
+
+### Worker setup
+
+#### Install dependencies
+
+`sudo apt install docker.io git`
+`sudo apt install bluez bluetooth`
+`sudo killall -9 bluetoothd`
+    * daemon cannot be running when docker starts
+    * you can change AutoEnable=true to false in /etc/bluetooth/main.conf but it still needs to start at least once
+
+#### Install worker
+
+`sudo apt install lava-dispatcher`
+`sudo vim /etc/lava-dispatcher/lava-worker`
+    * set WORKER_NAME and URL
+    * set the token to the string you get from adding a worker in the server
+
+#### Device access
+
+`sudo usermod -aG dialout $(whoami)`
+`git clone https://github.com/pyocd/pyOCD.git`
+`sudo cp pyOCD/udev/*.rules /etc/udev/rules.d`
+
+`sudo nano /etc/dbus-1/system.d/bluetooth.conf`
+    * copy the bluetooth policy from root to lava-admin
+
+#### Create workspace
+
+`sudo mkdir -p /opt/lava-worker`
+`sudo chmod 777 /opt/lava-worker`
+`cd /opt/lava-worker`
+`git clone https://github.com/paul-szczepanek-arm/lava-prototype lava-scripts`
+
+## Running LAVA server docker compose
+
+Instead of installing your lava server you may use docker compose provided by us.
+
+- `git clone https://github.com/ARMmbed/mbed-os-lava-docker-compose`
+-  Create a certificate and key and replace the ones in apache2/cert (you can try the free cert from
+   [zerossl](https://app.zerossl.com/certificate/new))
+- `docker-compose build lava-server`
+- `docker-compose up lava-server`
+
+Website is available on localhost, username and passowrd are `lava-admin`. Change the password after logging in.
+
+## Running a premade SD card image with the worker on your RPi
+
+Download the SD card image [MISSING LINK](http://) and copy it direclty to the root of the sd card device with `dd`.
```

Comment:
I would suggest use of raspberry pi imager: https://www.raspberrypi.org/software/ .